README.md

```diff
@@ -22,29 +22,20 @@ They also need a page to show names that should be changing within 28 days. The
 Keep in mind there are restrictions on what a citizen can name themselves:
 1. They cannot re-use a name that they've had in the past
 1. They cannot use a name that someone else currently has
- 
-##### Create any necessary pages that allow government officials to:
-1. Log in/out
-1. View the names of all citizens
-1. View any particular citizen to:
-  1. See their current name
-  1. See their past names
-  1. Submit a new name on their behalf
 
 ### Solution
 For your solution we expect to see:
 1. Usage of a relational database such as PostgreSQL
-1. A Rest or GraphQL API written in Laravel
+1. A Rest or GraphQL API
 1. A frontend built in JavaScript, ideally ES6/TypeScript/React, that performs calls to your API
   1. You are not expected to create a fancy looking page. Feel free to use a basic CSS framework or none at all
-1. The database, backend, and frontend to be in Docker containers
+1. Optional: The database and app to be in Docker containers
```

Comment:
I think the frontend can still be in a container if they want, but it might just be easier to remove this line completely